src/page_components/listing/ListingDetailHeader.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react"
+import { BADGES } from "../../../.storybook/constants"
+
+import { ListingDetailHeader } from "./ListingDetailHeader"
+
+export default {
+  title: "Listing/ListingDetailHeader 🚩",
+  parameters: {
+    badges: [BADGES.GEN2],
+  },
+}
+
+export const Default = () => {
+  return (
+    <div style={{ maxWidth: "500px" }}>
+      <ListingDetailHeader
+        imageAlt={"Building"}
+        imageSrc="https://housing.acgov.org/images/listing-eligibility.svg"
```

Comment:
should we consider uploading our own image to cloudinary and using it instad of using acgov site's images?

Reply:
Updated!